x-pack/platform/plugins/shared/triggers_actions_ui/kibana.jsonc

```diff
@@ -26,6 +26,7 @@
       "dataViewEditor",
       "alerting",
       "actions",
+      "dashboard",
```

Comment:
@fkanout We can [safely](https://github.com/elastic/kibana/pull/225295/files#r2193255102) remove this dependency.